numpy/core/tests/test_function_base.py

```diff
@@ -301,9 +301,9 @@ def __mul__(self, other):
     def test_denormal_numbers(self):
         # Regression test for gh-5437. Will probably fail when compiled
         # with ICC, which flushes denormals to zero
-        for dt in (dtype(f) for f in typecodes['Float']):
-            stop = finfo(dt).tiny * finfo(dt).resolution
-            assert_(any(linspace(0, stop, 10, endpoint=False, dtype=dt)))
+        for ftype in sctypes['float']:
+            stop = nextafter(ftype(0), ftype(1)) * 5  # A denormal number
```

Comment:
Testing for a bug in `nextafter` sounds like something that should be tested somewhere, and if not elsewhere, here is as good a place as any